content/agent/faq/why-don-t-i-see-the-system-processes-open-file-descriptors-metric.md

```diff
@@ -3,19 +3,71 @@ title: Why don't I see the 'system.processes.open_file_descriptors' metric?
 kind: faq
 ---
 
-Users that enable the [process check][1] in a Datadog Agent running on Linux OSs notice that the `system.processes.open_file_descriptors` metric is not collected or reported by default. This occurs when the process being monitored by the process check runs under a different user than the Agent - the `dd-agent` user doesn't have full access to all files in `/proc`, which is where the Agent looks to collect data for this metric.
+Users that enable the [process check][1] in a Datadog Agent running on a Linux OS notice that the `system.processes.open_file_descriptors` metric is not collected or reported by default. This occurs when the process being monitored by the process check runs under a different user than the Agent (`dd-agent`). The user doesn't have full access to all files in `/proc`, which is where the Agent looks to collect data for this metric.
+
+{{< tabs >}}
+{{% tab "Agent v6.3+" %}}
 
 Users can enable the `try_sudo` option in the process check configuration and add the appropriate sudoers rules:
+
 ```
 dd-agent ALL=NOPASSWD: /bin/ls /proc/*/fd/
 ```
+
 This allows the process check to use `sudo` to execute the `ls` command but only to the list of contents of the path `/proc/*/fd/`.
 
 If users see this line in the Datadog error.log: `sudo: sorry, you must have a tty to run sudo`, users should visudo and comment out the line `Default requiretty`.
 
-See the following Github issues for more info on this matter as well as other potential methods of capturing this metric on Linux machines.
+{{% /tab %}}
+{{% tab "Agent v6" %}}
+
+If you are running Agent v6 less than 6.3, we recommend updating the Agent and using the `try_sudo` option. If you are unable to update, a workaround for this issue is running the Agent as `root`.
+
+**NOTE**: We do not recommend running the Agent as `root`. This isn't specific to the Datadog Agent or due to any concern that something untrustworthy is happening in any way. Instead, we don't recommend running the daemon as `root` as this is best practice for most processes on Linux. If you have any personal cause for concern, the Agent is open source and may be audited by you or your team via the [GitHub repository][2].
```

Comment:
Subjective, but i think just recommending not to run the Agent as `root` is sufficient without the explanation.

Reply:
It's important to explain that we don't recommend running the agent as root because of Linux best practices and state the agent doesn't do any malicious. This language is consistent with messages sent to customers in support tickets.